src/librustc_trans/partitioning.rs

```diff
@@ -280,75 +280,74 @@ fn place_root_translation_items<'a, 'tcx, I>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
     let mut internalization_candidates = FxHashSet();
 
     for trans_item in trans_items {
-        let is_root = trans_item.instantiation_mode(tcx) == InstantiationMode::GloballyShared;
```

Comment:
This file's diff is probably best viewed with [no whitespace](https://github.com/rust-lang/rust/pull/45075/files?w=1#diff-1b0042d63997a28b01473b2fba63406a) (very few changes here, just indentation)